osrm/core.py

```diff
@@ -221,9 +221,9 @@ def simple_route(coord_origin, coord_dest, coord_intermediate=None,
         elif geometry in ("polyline", "geojson") and output == "routes":
             return parsed_json["routes"]
         else:
-            if geometry == "wkb":
+            if geometry.lower() == "wkb":
                 func = Geometry.ExportToWkb
-            elif geometry == "wkt":
+            elif geometry.lower() == "wkt":
```

Comment:
Could you also correct it [here](https://github.com/ustroetz/python-osrm/blob/master/osrm/core.py#L497)?